### PR TITLE
[No JIRA] Increase tap area for iOS nav bar buttons

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     zIndex: 2,
     alignItems: 'center',
+    height: '100%',
```
Comment (georgegillams) [inline packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js]: Doesn't this need `justifyContent: 'center'` too?

File: packages/react-native-bpk-component-icon/package-lock.json [hunk 2]
```diff
@@ -17,12 +17,6 @@
 				"color": "^3.0.0"
 			}
 		},
-		"brcast": {
```
Comment (georgegillams) [inline packages/react-native-bpk-component-icon/package-lock.json]: Why did `package-lock.json` change?

File: packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js (hunk 1 above)
Comment (k0nserv) [reply packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js]: Nope, it's a row and already has `aligntItems: 'center'`

File: packages/react-native-bpk-component-icon/package-lock.json (hunk 2 above)
Comment (k0nserv) [reply packages/react-native-bpk-component-icon/package-lock.json]: No idea

File: packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js [hunk 3]
```diff
@@ -40,16 +40,21 @@ const styles = StyleSheet.create({
...
   },
   backIcon: {
     marginEnd: 8, // eslint-disable-line backpack/use-tokens
   },
+  backIconWithoutTitle: {
+    marginEnd: 16, // eslint-disable-line backpack/use-tokens
```
Comment (georgegillams) [inline packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js]: What's the reason for not using tokens? I think there should be a comment to go with this explaining the decision...

Comment (k0nserv) [reply packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js]: Sorted

File: packages/react-native-bpk-component-icon/package-lock.json (hunk 2 above)
Comment (georgegillams) [reply packages/react-native-bpk-component-icon/package-lock.json]: Do you wanna revert it then? 😆

Comment (shaundon) [reply packages/react-native-bpk-component-icon/package-lock.json]: Yeah we should probably revert this

**Pull request merged**